src/test/resources/application-test.yml

```diff
@@ -8,6 +8,11 @@ spring:
   datasource:
     url: 'jdbc:h2:mem:testdb;Mode=Oracle'
     type: com.zaxxer.hikari.HikariDataSource
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          jwk-set-uri: http://localhost:9010/.well-known/jwks.json
```

Comment:
Using `jwk-set-uri` in the test config was a hack on my part so the mock server didn't have to implement `/auth/issuer`. Ideally the mock server should respond to `/auth/issuer` as well as `/.well-known/jwks.json` to be like we do in the runtime configuration, and then `issuer-uri` could be used here to make the tests more realistic. Just an observation :D